translations/ggdm30.js

```diff
@@ -2422,6 +2443,23 @@ ggdm30 = {
           // Validate attrs: remove all that are not supposed to be part of a feature
           ggdm30.validateAttrs(geometryType,returnData[i]['attrs'],notUsedTags,transMap);
 
+          // If we have unused tags, add them to the memo field
+          if (Object.keys(notUsedTags).length > 0 && ggdm30.config.OgrNoteExtra == 'attribute')
+          {
+            var tStr = '<OSM>' + JSON.stringify(notUsedTags) + '</OSM>';
+
+            // there is always one.....
+            // ZI031 (Dataset) has MEM instead of ZI006_MEM.  Why???????
```

Comment:
Why do they do this????

Reply:
If they didn't, translations wouldn't require any effort at all!